deals/deals_test.go

```diff
@@ -42,10 +43,25 @@ func TestStore(t *testing.T) {
 	for _, nm := range numMiners {
 		t.Run(fmt.Sprintf("CantMiners%d", nm), func(t *testing.T) {
 			client, _, _ := tests.CreateLocalDevnet(t, nm)
-			m, err := New(client, WithImportPath(filepath.Join(tmpDir, "imports")))
+			m, err := New(tests.NewTxMapDatastore(), client, WithImportPath(filepath.Join(tmpDir, "imports")))
 			checkErr(t, err)
-			_, err = storeMultiMiner(m, client, nm, randomBytes(600))
+			_, pcids, err := storeMultiMiner(m, client, nm, randomBytes(600))
 			checkErr(t, err)
+			pending, err := m.PendingDealRecords()
+			require.Nil(t, err)
+			require.Len(t, pending, nm)
+			final, err := m.FinalDealRecords()
+			require.Nil(t, err)
+			require.Empty(t, final)
+			err = waitForDealComplete(client, pcids)
+			checkErr(t, err)
+			time.Sleep(util.AvgBlockTime)
```

Comment:
Not sure how to avoid this sleep. It exists because our deal monitoring uses `Watch` which polls for the deal state every `util.AvgBlockTime`. So we need to wait at least that long here to know that we've detected the deal state in `Watch`. Unfortunate for devnet because `util.AvgBlockTime` is much longer than needed since it is the average for test net or main net. Ideas?

Reply:
For tests, in some `TestMain`, you could do `util.AvgBlockTime=smaller_value` so things happen more often.
I do this in `integration_test.go` in `TestMain()`. Would that help?

Reply:
Ah yea, that would work.